fastparquet/test/test_util.py

```diff
@@ -1,28 +1,50 @@
 import pytest
 
-from fastparquet.util import analyse_paths, get_file_scheme, val_to_num
+from fastparquet.util import analyse_paths, get_file_scheme, val_to_num, join_path
 
 
 def test_analyse_paths():
     file_list = ['a', 'b']
-    base, out = analyse_paths(file_list, '/')
+    base, out = analyse_paths(file_list)
     assert (base, out) == ('', ['a', 'b'])
 
     file_list = ['c/a', 'c/b']
-    base, out = analyse_paths(file_list, '/')
+    base, out = analyse_paths(file_list)
     assert (base, out) == ('c', ['a', 'b'])
 
     file_list = ['c/d/a', 'c/d/b']
-    base, out = analyse_paths(file_list, '/')
+    base, out = analyse_paths(file_list)
     assert (base, out) == ('c/d', ['a', 'b'])
 
+    file_list = ['/c/d/a', '/c/d/b']
+    base, out = analyse_paths(file_list)
+    assert (base, out) == ('/c/d', ['a', 'b'])
+
     file_list = ['c/cat=1/a', 'c/cat=2/b', 'c/cat=1/c']
-    base, out = analyse_paths(file_list, '/')
+    base, out = analyse_paths(file_list)
     assert (base, out) == ('c', ['cat=1/a', 'cat=2/b', 'cat=1/c'])
 
-    file_list = ['c\\cat=1\\a', 'c\\cat=2\\b', 'c\\cat=1\\c']
```

Comment:
As it stands, you can still provide windows paths to ParquetFile, and they will get passed through, no?

Reply:
Yes, you can still provide windows paths to ParquetFile.  The `tempdir` fixture provides windows paths during testing, and the test pass on windows.

I added back this test, with minor changes, to show that windows paths will work too.